service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -1,23 +1,94 @@
 package uk.gov.hmcts.reform.fpl.model;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Builder;
 import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentSocialWorkOther;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.EPOGroup;
 
 import java.util.List;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
 @Data
-@Builder(toBuilder = true)
-@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
You might want to bring this back. Without it any data model change will require change in `CaseData` even if it is not needed in any Java code. 

Usually I am against ignoring properties but in context of how we use data classes it might be better to ignore it for some time.